### ISSUE TITLE
aria-label cannot be set correctly if it contains quotes

### ISSUE THREAD

**Issue by mkoryak** [mkoryak/floatThead#440]
because I am not setting attributes but doing it the ghetto jQuery way. 
Fix that and also make setting aria labels more flexible. See #439.
